interfaces/builtin/pluggable_storage.go

```diff
@@ -0,0 +1,65 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const pluggableStorageConnectedPlugAppArmor = `
+# Description: Can mount and unmount removable storage. This is restricted
+# because it gives privileged access to mount commands and should only be used
+# with trusted apps.
+# Usage: reserved
+
+# Needed for mount/unmount operations
+capability sys_admin,
+
+# TODO: These should be confined by path, need to come up with reasonable
```

Comment:
To this point, I have a snap of udisks2 that uses /run/media/<user>/ as that mount point (since /media is in the read-only filesystem)